clientgui/BOINCGUIApp.cpp

```diff
@@ -108,6 +142,13 @@ bool CBOINCGUIApp::OnInit() {
 #if SUPPORTDARKMODE
     wxSystemAppearance appearance = wxSystemSettings::GetAppearance();
     m_isDarkMode = appearance.IsDark();
+// Dark theme only makes sense on Win if WxWidgets has version >3.3
+#if defined(__WXMSW__) && wxCHECK_VERSION(3,3,0)
+    // wxWidgets' IsDark() returns false on Win 11 now even if dark mode is enabled
+    // so we need to additionally check the registry for the dark mode setting
+    m_isDarkMode = m_isDarkMode || IsWindowsInDarkMode();
+    MSWEnableDarkMode(wxApp::DarkMode_Auto);
```

Comment:
Are you sure this is needed? I do remember when I tested the dark mode, I didn't have this enabled